run-your-own-gem-server.md

```diff
@@ -14,31 +14,36 @@ may have private code, internal to your organization, that you'd like to
 distribute and manage as gems without making the source publicly available.
 
 There are a few options to set up a server to host gems from within your
-organization. This guide covers the `gem server` command and the [Gem in a
-Box](https://github.com/geminabox/geminabox) project. It also discusses how to
-use these servers as gem sources during development.
+organization. This guide covers the [Gemstash](https://github.com/rubygems/gemstash)
+and [Gem in a Box](https://github.com/geminabox/geminabox) project. It also
+discusses how to use these servers as gem sources during development.
 
-## Running the built-in gem server
+## Running Gemstash
 
-When you install RubyGems, it adds the `gem server` command to your system.
-This is the fastest way to start hosting gems. Just run the command:
+If you're looking for ways to host your own private gem servers, try Gemstash. 
+Gemstash is a Rubygems.org gem server application maintained by RubyTogether.
```

Comment:
Probably first line form gemstash readme is more accurate:
```
Gemstash is both a cache for remote servers such as https://rubygems.org, and a private gem source.
```

Reply:
I revised the entire section about Gemstash a bit.